src/index.js

```diff
@@ -173,17 +272,67 @@ const renderLevel = (root, levelNumber) => {
 
     if (soko.goto(row, col)) {
       renderBoard();
+
+      if (soko.solved()) {
+        const score = localStorage.getItem(levelNumber+1);
+        if (score) {
+          if (score > soko.movesSize()){
+            updateBestScore(levelNumber+1, soko.movesSize());
+          }
+        } 
+        else if (!score) {
+          updateBestScore(levelNumber+1, soko.movesSize());
+        }
+        renderLevelComplete();
+      }
     }
   });
+
+
+
+ updateStatus = () => {
+    document.getElementById("level-label").innerHTML = `${soko.levelNumber()+1}`;
```

Comment:
```suggestion
    document.getElementById("level-label").innerHTML = `${soko.levelNumber() + 1}`;
```

Same formatting elsewhere.